src/server/fetch/fetch.js

```diff
@@ -0,0 +1,30 @@
+import fetch from 'node-fetch';
+import nodeLogger from '#lib/logger.node';
+import { BFF_FETCH_ERROR } from '#lib/logger.const';
+import getAgent from './agent';
+
+const logger = nodeLogger(__filename);
+
+const fetchWithCerts = async (requestUrl, service) => {
+  const agent = await getAgent();
+
+  const opts = {
+    method: 'GET',
+    agent,
+    headers: {
+      Accept: 'application/json',
+    },
+  };
+
+  try {
+    return await fetch(requestUrl, opts);
+  } catch (error) {
+    logger.error(BFF_FETCH_ERROR, {
+      service,
+      error,
+    });
+    return null;
```

Comment:
I think we are better allowing the error to propogate here, I think it would fail here where it would try destructure null and fail causing a duplicate error log: https://github.com/bbc/simorgh/pull/9813/files#diff-6e0488cd2114bbba672b980609882f2d3f360a3cedb6732d5e2d5bb89faaf361R9 

Can we have tests added for there error scenarios, i.e. mocking fetch throwing an error in this scenario?